frontend/src/components/Wheel/index.js

```diff
@@ -15,7 +15,7 @@ const Wheel = ({ email }) => {
   async function startRotate() {
     setIsRotating((prev) => !prev);
 
-    await fetch(`${process.env.REACT_APP_BACKEND_PATH}/win-prize`, {
+    await fetch(`${process.env.REACT_APP_BACKEND_PATH}/prize`, {
```

Comment:
This needs to exist in the backend.

Reply:
In your Rails server console:

```
Started POST "/prize" for ::1 at 2022-08-24 11:18:56 +0200
  
ActionController::RoutingError (No route matches [POST] "/prize"):
```

Reply:
And in your browser console.

<img width="533" alt="Screenshot 2022-08-24 at 11 22 52" src="https://user-images.githubusercontent.com/26620750/186382005-d1e6ee1b-fd91-4fba-9c98-ba875efd4434.png">

Reply:
You can either also change the backend route to `/prize` or you can keep the backend route as `/win-prize`

Reply:
I think I fixed with [this](https://github.com/Subvisual-Academy/subvisuwheel/pull/164/commits/c60ae90a8a5e6072899466c929c849a4fff79462) just before you commented or am I wrong. Got confused 😄

Reply:
You're right!